<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCommentDialog</class>
 <widget class="QWidget" name="AddCommentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить комментарий</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="edtPharmacyComment"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lblPharmacy">
     <property name="text">
      <string>Для аптеки:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="lblRecommendRLS">
     <property name="text">
      <string>Рекомендованный РЛС:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="CRLSComboBox" name="cmbRecommendRLS"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRLSComboBox</class>
   <extends>QComboBox</extends>
   <header>library/RLS/RLSComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
